mac

```diff
@@ -24,29 +24,41 @@ append_to_file() {
     else
       file="$HOME/.zshrc"
     fi
-  elif [ "$file" = "$HOME/.bash_profile" ]; then
-    file="$HOME/.bash_profile"
   fi
 
   if ! grep -qs "^$text$" "$file"; then
     printf "\n%s\n" "$text" >> "$file"
   fi
 }
 
-create_zshrc_and_set_it_as_shell_file() {
-  if [ ! -f "$HOME/.zshrc" ]; then
-    touch "$HOME/.zshrc"
+append_to_posix_shell_file() {
+  case "$SHELL" in
+    */fish) :
+      fancy_echo "Not appending %s to config.fish" "$1"
```

Comment:
Out of curiosity, why are we not writing stuff to the fish file?

Reply:
Any lines calling `append_to_posix_shell_file` are invalid syntax for fish, but appeared to be valid for both `zsh` and `bash`. Some places the fish defaults actually handle what we wanted to do (like adding `/usr/local/bin` to `PATH`) and in others I used `append_to_file "$shell_file"...` to send the correct line to `config.fish`